src/aap_eda/core/utils/credentials.py

```diff
@@ -108,7 +111,10 @@ def validate_inputs(schema: dict, inputs: dict) -> dict:
                 inputs=inputs,
             )
             if bool(result):
-                errors[display_field] = result
+                if PROTECTED_PASSPHRASE_ERROR in result:
+                    errors["inputs.ssh_key_unlock"] = result
```

Comment:
@hsong-rh do we have to check if the customer provided inputs has the ssh_key_unlock? Or would that validation kick in only when both keys are present.

Reply:
@mkanoor Yes, we check them at https://github.com/ansible/eda-server/blob/main/src/aap_eda/core/utils/credentials.py#L332

Reply:
@mkanoor We will handle your scenario later: create a credential type without `Private Key Passphrase` field. In that case, we may show a warning message.